tests/e2e/specs/wordpress/quickEdit.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import {
+  loginUser,
+  switchUserToAdmin,
+  visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+/**
+ * Internal dependencies
+ */
+import { createNewStory } from '../../utils';
+
+describe('Quick Edit', () => {
+  beforeAll(async () => {
+    await loginUser('author', 'password');
+  });
+
+  afterAll(async () => {
+    await switchUserToAdmin();
+  });
+
+  it('save quick edit', async () => {
+    await createNewStory();
+
+    await expect(page).toMatchElement('input[placeholder="Add title"]');
+
+    const storyTitle = 'Test quick edit';
+
+    await page.type('input[placeholder="Add title"]', storyTitle);
+
+    // Publish story.
+    await expect(page).toClick('button', { text: 'Publish' });
+
+    await expect(page).toMatchElement('button', { text: 'Dismiss' });
+
+    await visitAdminPage('edit.php', 'post_type=web-story');
+
+    await expect(page).toMatch(storyTitle);
+
+    // Make row actions appear.
+    const elmId = await page.evaluate((filename) => {
+      return document
+        .querySelector(`a[aria-label="“${filename}” (Edit)"]`)
+        .closest('tr').id;
+    }, storyTitle);
+
+    await page.hover(`#${elmId}`);
+
+    await expect(page).toClick(
+      `button[aria-label="Quick edit “${storyTitle}” inline"]`
+    );
+
+    await page.type('input[name="post_title"]', ' - updated.');
+    await expect(page).toClick('button', { text: 'Update' });
+
+    const storyTitleNew = 'Test quick edit – updated.';
+
+    await expect(page).toMatch(storyTitleNew);
+    await Promise.all([
+      page.waitForNavigation(),
+      await expect(page).toClick(`a[aria-label="View “${storyTitleNew}”"]`),
+    ]);
+
+    await expect(page).not.toMatch('amp=""');
```

Comment:
@swissspidy Should I add a percy screenshot here?

Reply:
I don't think this is a good assertion for this test.

If we have a better assertion (for example look for `amp-story` element—this would be removed without KSES), then we don't necessarily need a screenshot

Reply:
I still think this not an ideal assertion, marking as unresolved.